server/src/main/java/org/opensearch/cluster/routing/allocation/allocator/RemoteShardsBalancer.java

```diff
@@ -266,6 +266,15 @@ void balance() {
         }
     }
 
+    /**
+     * Intended to trigger shard relocation for the shards that are submitted
+     * for tiering to go from remote node pool to local capable node
+     * pool.
+     * yet to be implemented
+     */
+    @Override
+    void tierShards() {}
```

Comment:
nit: should we throw `UnsupportedOperationException` from here as W2H is not supported yet?